pkg/clients/clients.go

```diff
@@ -91,23 +94,40 @@ func New(cc *controllercmd.ControllerContext) (*Clients, error) {
 		return nil, err
 	}
 
+	infFact := dynamicinformer.NewDynamicSharedInformerFactory(dynClient, defaultResyncPeriod)
+
+	clusterExtensionGVR := schema.GroupVersionResource{
+		Group:    ocv1alpha1.ClusterExtensionGVK.Group,
+		Version:  ocv1alpha1.ClusterExtensionGVK.Version,
+		Resource: "clusterextensions",
+	}
```

Comment:
Pretty sure this GVK variable is already gone from upstream. I'd suggest going ahead and changing this to the following:
```suggestion
 	clusterExtensionGVR := ocv1alpha1.GroupVersion.WithResource("clusterextensions")
```